exporter/collector/metrics.go

```diff
@@ -245,7 +245,17 @@ func (me *MetricsExporter) PushMetrics(ctx context.Context, m pmetric.Metrics) e
 			s, _ := status.FromError(err)
 			st = statusCodeToString(s)
 
-			recordPointCountDataPoint(ctx, len(ts), st)
+			summaryPointCount := len(ts)
+			for _, detail := range s.Details() {
+				if summary, ok := detail.(*monitoringpb.CreateTimeSeriesSummary); ok {
+					// if success != total, we're reporting a failed create and change to using the diff
+					if summary.TotalPointCount != summary.SuccessPointCount {
+						summaryPointCount = int(summary.TotalPointCount - summary.SuccessPointCount)
+					}
+				}
+			}
+
+			recordPointCountDataPoint(ctx, summaryPointCount, st)
```

Comment:
can we always record the number of successful points, even if a few failed?  It would be useful for creating an SLO, for example.

Reply:
trying to add this I actually caught that `s.Details()` isn't returning anything (at least in our test cases). it was just passing because the total was only `1`. so I need to look at this more, but I will add that as part of it